### ISSUE TITLE
Feature request: Iframe debugging

### ISSUE THREAD

**Issue by realappie** [rangle/augury#1195]
First of all, thanks for your great work on augury Its a perfect tool for debugging angular applications. 

Im currently building an application where I have a second angular application running in an iframe, I cannot use augury to debug the iframe. Even if I switch to it in here.

![](https://image.prntscr.com/image/J_XJqeZfQdms4zJ1JDzxUQ.png)

Being able to debug iframes would be a great addition to augury.

**Comment by frsauvage** [comment]
I agree : I cannot debug my angular vsphere plugin with augury, cause inside a vsphere iframe ...

**Comment by pirkeyj** [comment]
Did this issue ever resolved? I'm having the same issue as well.  Our angular app has to be hosted within a vendor's iFrame and I can't seem to get Augury to function.  Unfortunately, our application relies on that iFrame to function so bypassing the iframe doesn't solve the issue.  I am very new to Augury so maybe I'm missing something?

**Comment by realappie** [comment]
I ended up uninstalling augury & just use `ng.probe($$('COMPONENT_NAME')[0]).componentInstance` nowadays.

**Comment by eladeyal-intel** [comment]
@sumitarora can we help triage this?
